components/accept-terms.stories.js

```diff
@@ -56,12 +55,12 @@ Basic.args = {};
 
 export const NewBuyFlow = (args) => <AcceptTerms {...args} />;
 NewBuyFlow.args = {
-	isAuthFirstAccount: true,
+	isAccessPage: true,
```

Comment:
Did you miss changing these props?
```suggestion
	withPrivacyPolicyTerms: true,
```

Same below